examples/device/ep/tests/control/control.py

```
@@ -0,0 +1,489 @@
+
+    common_kwargs = dict(
+        rank=global_rank,
+        num_ranks=num_ranks,
+        other_ranks=other_ranks,
+        tcp_store=tcp_store,
+        disable_ll_nvlink=args.disable_ll_nvlink,
+        num_experts_per_rank=args.num_experts_per_rank,
+        num_rdma_bytes=num_rdma_bytes,
+        warmup=args.warmup,
+        iters=args.iters,
+    )
```

Ruff (0.15.9)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)